errors = {
    "traceback": "Traceback (most recent call last):",
    "no_traceback": "No traceback available.",
    "file": "File \"{filename}\", line {line}, in {name}",
    "event_invalid": "The event {event} doesn't exist",
    "function_invalid": "{function_name}() has invalid args for this event. The correct way:\ndef {function_name}({arguments}): ...",
    "bot_is_already_running": "The bot is already running",
    "bot_is_running": "The bot is currently active and you cannot change anything",
    "request_failed": "The HTTP request code is {code} with error \"{error}\"",
    "dispy_request_error": "An error occured with dispy during HTTP request: {error}"
}